packages/sd-jwt/src/verify.ts

```diff
@@ -0,0 +1,113 @@
+import * as jwt from "jsonwebtoken";
+import * as jose from "jose";
+import * as jwks from "jwks-rsa";
+import { SdJwt } from "./types";
+
+type VerifyOptions = { jwksUri: string };
+type VerifyResult = SdJwt;
+
+function isSupportedAlgorithm(alg: string): alg is jwt.Algorithm {
```

Comment:
nit: having a `const ALGS = new Set([... ])` or an object would be faster

Reply:
done in https://github.com/pagopa/io-wallet/pull/4/commits/25dd9ccbfc17ba01a06d004e96eee37d97b5d851